[2025-02-19T11:33:40.035+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T11:33:40.046+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.parser_for_Eapteka manual__2025-02-19T11:33:38.061925+00:00 [queued]>
[2025-02-19T11:33:40.055+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.parser_for_Eapteka manual__2025-02-19T11:33:38.061925+00:00 [queued]>
[2025-02-19T11:33:40.058+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-19T11:33:40.194+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): parser_for_Eapteka> on 2025-02-19 11:33:38.061925+00:00
[2025-02-19T11:33:40.202+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'parser_for_Eapteka', 'manual__2025-02-19T11:33:38.061925+00:00', '--job-id', '895', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpi_cagoxq']
[2025-02-19T11:33:40.200+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=212) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T11:33:40.204+0000] {standard_task_runner.py:72} INFO - Started process 221 to run task
[2025-02-19T11:33:40.204+0000] {standard_task_runner.py:105} INFO - Job 895: Subtask parser_for_Eapteka
[2025-02-19T11:33:40.238+0000] {task_command.py:467} INFO - Running <TaskInstance: pipeline.parser_for_Eapteka manual__2025-02-19T11:33:38.061925+00:00 [running]> on host 23903a4a7553
[2025-02-19T11:33:40.308+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='parser_for_Eapteka' AIRFLOW_CTX_EXECUTION_DATE='2025-02-19T11:33:38.061925+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-19T11:33:38.061925+00:00'
[2025-02-19T11:33:40.311+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-19T11:33:40.311+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-19T11:33:40.312+0000] {logging_mixin.py:190} INFO - Current task name:parser_for_Eapteka state:running start_date:2025-02-19 11:33:40.046427+00:00
[2025-02-19T11:33:40.312+0000] {logging_mixin.py:190} INFO - Dag name:pipeline and current dag run status:running
[2025-02-19T11:33:40.312+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T11:33:40.313+0000] {parser_1.py:27} INFO - Open webdriver_remote
[2025-02-19T11:33:40.837+0000] {parser_1.py:39} INFO - Webdriver opened
[2025-02-19T11:34:47.270+0000] {parser_1.py:63} ERROR - Error occurred: Message: unknown error: net::ERR_NAME_NOT_RESOLVED
  (Session info: chrome=124.0.6367.78)
Stacktrace:
#0 0xaaaae5bb7d44 <unknown>
#1 0xaaaae58de374 <unknown>
#2 0xaaaae58d8a34 <unknown>
#3 0xaaaae58cc274 <unknown>
#4 0xaaaae58cd518 <unknown>
#5 0xaaaae58cc5e4 <unknown>
#6 0xaaaae58cbf64 <unknown>
#7 0xaaaae58cbe44 <unknown>
#8 0xaaaae58ca4bc <unknown>
#9 0xaaaae58ca7b8 <unknown>
#10 0xaaaae58e13ec <unknown>
#11 0xaaaae5958848 <unknown>
#12 0xaaaae5958400 <unknown>
#13 0xaaaae59114a4 <unknown>
#14 0xaaaae59120e0 <unknown>
#15 0xaaaae5b85ac0 <unknown>
#16 0xaaaae5b88a64 <unknown>
#17 0xaaaae5b88630 <unknown>
#18 0xaaaae5b76d6c <unknown>
#19 0xaaaae5b89128 <unknown>
#20 0xaaaae5b61010 <unknown>
#21 0xaaaae5ba8550 <unknown>
#22 0xaaaae5ba8758 <unknown>
#23 0xaaaae5bb7250 <unknown>
#24 0xffff8cacee58 <unknown>
#25 0xffff8cb37f9c <unknown>

[2025-02-19T11:34:47.369+0000] {python.py:240} INFO - Done. Returned value was: Empty DataFrame
Columns: []
Index: []
[2025-02-19T11:34:47.410+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T11:34:47.411+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=parser_for_Eapteka, run_id=manual__2025-02-19T11:33:38.061925+00:00, execution_date=20250219T113338, start_date=20250219T113340, end_date=20250219T113447
[2025-02-19T11:34:47.429+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-19T11:34:47.429+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-19T11:34:47.430+0000] {logging_mixin.py:190} INFO - Dag name:pipeline queued_at:2025-02-19 11:33:38.080941+00:00
[2025-02-19T11:34:47.430+0000] {logging_mixin.py:190} INFO - Task hostname:23903a4a7553 operator:_PythonDecoratedOperator
[2025-02-19T11:34:47.446+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T11:34:47.479+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-19T11:34:47.480+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
